### PR TITLE
added invitations for group method & naming refactoring

### PR DESCRIPTION
**Pull request by 4KevR**
This PR implements:

- invitations for group endpoint
- naming adjustments
- updated postman collection

### CODE REVIEW

**Review by memeToasty — APPROVED** [review]

**Review by Julian702 — APPROVED** [review]

**Review by TomRomeo — COMMENTED** [review]
Looks good, I am just uncertain about a few things:

File: backend/store/postgres.go [hunk 1]
```diff
@@ -283,6 +283,13 @@ func (s *CollaborationStoreImpl) FindGroupInvitationsByUserID(userID string) (gr
 	return
 }
 
+func (s *CollaborationStoreImpl) FindGroupInvitationsByGroupID(groupID string) (groupAdmissions []model.GroupAdmission, err error) {
```
[the remark this answers is not in the document]
Comment (4KevR) [reply backend/store/postgres.go:300]: added in 3eb202a3ea6af5d33090b53c0debfcd65ffcf732

[the remark this answers is not in the document]
Comment (4KevR) [reply backend/store/postgres.go:300]: The slice also leads to problems related to the `ConvertToTypeArray` function. Will be addressed in a future refactoring PR.

File: backend/services/collaboration/handler/collaboration.go [hunk 2]
```diff
@@ -53,6 +53,27 @@ func (e *Collaboration) checkUserRoleAccess(_ context.Context, userID string, gr
 	return nil
 }
 
+func (e *Collaboration) generateGroupMemberAdmissionResponse(ctx context.Context, groupAdmissions []model.GroupAdmission) (memberAdmissions []*pb.MemberAdmission, err error) {
```
[the remark this answers is not in the document]
Comment (TomRomeo) [reply backend/services/collaboration/handler/collaboration.go:59]: Let's tackle that problem when we will rewrite and simplify the proto logic 😪👍🏼

**Review by TomRomeo — APPROVED** [review]
Great work! ♥

**Pull request merged**